src/makeRobot.js

```diff
@@ -37,8 +38,76 @@
  *
  * @return {Robot}
  */
+
 function makeRobot(name, wheels, version) {
   // write code here
+  const Robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
shorthand writing looks like this (only when the names are the same)

```suggestion
    name,
    wheels,
    version,
```